docs/source/format/Glossary.rst

```diff
@@ -197,6 +197,10 @@ Glossary
 
        Not part of the columnar format; this term is specific to
        certain language implementations of Arrow (primarily C++ and
-       its bindings).
+       its bindings and Go).
+
+       .. image:: ../cpp/tables-versus-record-batches.svg
```

Comment:
For the record, is there any source file to this SVG (I don't know how it was generated)?

Reply:
The SVG is the source file; I wrote it by hand. It's just `rect` and `text` for the most part and I added the Apache license header as a comment.

Reply:
I didn't know people wrote SVG files by hand :-o